sympy/polys/numberfields.py

```diff
@@ -580,22 +580,29 @@ def _minpoly_compose(ex, x, dom):
 
 
 @public
-def minimal_polynomial(ex, x=None, **args):
+def minimal_polynomial(ex, x=None, compose=True, polys=False, domain=None):
     """
     Computes the minimal polynomial of an algebraic element.
 
     Parameters
     ==========
 
-    ex : algebraic element expression
-    x : independent variable of the minimal polynomial
+    ex : Expr
+        Element or expression whose minimal polynomial is to be calculated.
 
-    Options
-    =======
+    x : Symbol, optional
+        Independent variable of the minimal polynomial
 
-    compose : if ``True`` ``_minpoly_compose`` is used, if ``False`` the ``groebner`` algorithm
-    polys : if ``True`` returns a ``Poly`` object
-    domain : ground domain
+    compose : boolean, optional (default=True)
+        Method to use for computing minimal polynomial. If ``compose=True``
+        (default) then ``_minpoly_compose`` is used, if ``compose=False`` then
+        groebner bases are used.
```

Comment:
capitalize Groebner?

Reply:
It is capitalized in many places but in many more cases it is lower case. It's not important enough for a new cycle of tests. I'll commit as is.

Reply:
I missed this. Actually, I've mostly seen most of the usage in terms of capital 'G'. On wikipedia it is mostly used as 'Gröbner bases'.